cmd/internal/serverutil/main.go

```diff
@@ -238,3 +248,23 @@ func AnnounceSelf(ctx context.Context, client *clientv3.Client, etcdService, end
 		client.Revoke(ctx, leaseRsp.ID)
 	}
 }
+
+// listenKeepAliveRsp listens to `keepAliveRspCh` channel, and calls the cancel function
+// to notify the lease expired.
+func listenKeepAliveRsp(ctx context.Context, keepAliveRspCh <-chan *clientv3.LeaseKeepAliveResponse, cancel func()) {
```

Comment:
Do you think it'd be slightly easier to grok if this method blocked and the caller is responsible for doing `go listenKeepAliveRsp(...)`?

Reply:
fixed